econplayground/templates/assignment/assignment_step_layout.html

```diff
@@ -194,6 +200,9 @@
             stepQuestions[stepId] = questionId;
         }
     });
+    $('.assignment-grid select.ep-question').select2({
+        dropdownCSSClass: ':all:',
```

Comment:
Is this `':all:'` string intentional? I'm not sure what that's referring to - i assumed this should be `form-select` to give the select element bootstrap styles.